goonj/channels/email_channel.py

```diff
@@ -0,0 +1,52 @@
+import email.utils
+import smtplib
+from email.mime.text import MIMEText
+
+from goonj import core
+from goonj.channels.base_channel import BaseChannel
+from goonj.conf.constants import Sev
+
+
+class EmailChannel(BaseChannel):
+    """
+    Channel to send emails
+    """
+
+    def __init__(self, **kwargs):
+        """
+
+        :param kwargs:
+        dict of parameters required to initialze email channel,some field
+        are optional like subject
+        Initializing email channel using config
+
+        """
+        self.from_add = kwargs['from_add']
+        self.to = kwargs['to']
+        self.name = kwargs['name']
+        if 'subject' in kwargs:
+            self.subject = kwargs['subject']
+
+    def send(self, sev, message, subject=None, error_id=None, error=None,
+             tag_list=None):
```

Comment:
The how to send email should be injected in, cause a lot of ppl might already have email services, like we do with Notification service. Also, sending email in a sync way may not be the best idea. This should return instantly, cause email services are generally very slow.